protoc_plugin/test/protos/oneof.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+syntax = "proto3";
+
+package protobuf_unittest;
```

Comment:
Could you also add a oneof somewhere in the _leading_underscore.proto?

Reply:
No, since that one is `syntax = "proto2"`.

Reply:
Done